<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionSettingFrame</class>
 <widget class="QFrame" name="ActionSettingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="x1PosLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>69</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="absBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>35</y>
     <width>63</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>Abs</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="gxButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>69</y>
     <width>59</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>GX</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>109</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="gyButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>103</y>
     <width>58</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>GY</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="y1PosLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>103</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="speedEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>36</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QToolButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>Set=&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>285</y>
     <width>42</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>75</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ICComboBox" name="pointSel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>240</y>
     <width>136</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="relBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>238</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>Rel</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>271</y>
     <width>294</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMessageBox QAbstractButton
{
    min-width: 90px;
	min-height:30px;
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="gzButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>137</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="zPosLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>137</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>143</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="delayEdit">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>280</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>285</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="tPosLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>170</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>176</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="gPButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>204</y>
     <width>57</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>X2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="gtButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>59</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>GT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="pPosLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>204</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>285</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>285</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="curveBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
  <customwidget>
   <class>ICComboBox</class>
   <extends>QComboBox</extends>
   <header>iccombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
